bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java

```diff
@@ -459,15 +459,6 @@ public Type getType() {
         return type;
     }
 
-    @Override
-    public String getTitle() {
```

Comment:
If its required on field and panel, it should be part of AbstractBaseImpl ?

Reply:
it is required on formsContainer. but not on panel or any field. This is why putting the getTitle in AbstractBaseImpl either added title to json of all the component or removed from all the component.
Hence I had to move it down a hierarchy.

PanelImpl and AbstractFieldImpl with JsonIgnore and FormContainer without JsonIgnore.

Reply:
Panel/Field only have label and form has a title. Adding this function is confusing. Either make it as part of utils or use getLabel directly in ComponentDataImpl

Reply:
On AbstractFormComponent, check for instanceof and use API's accordingly. Adding getTitle just to make it uniform across all fields/panel/formcontainer is not the solution

Reply:
understood